apiserver/facades/client/applicationoffers/applicationoffers.go

```diff
@@ -157,6 +163,23 @@ func (api *OffersAPI) ListApplicationOffers(filters params.OfferFilters) (params
 	return result, nil
 }
 
+// ListApplicationOffers gets deployed details about application offers that match given filter.
+// The results contain details about the deployed applications such as connection count.
+func (api *OffersAPIV4) ListApplicationOffers(filters params.OfferFilters) (params.QueryApplicationOffersResultsV4, error) {
+	res, err := api.OffersAPI.ListApplicationOffers(filters)
+	if err != nil {
+		return params.QueryApplicationOffersResultsV4{}, errors.Trace(err)
+	}
+	var resultsV4 []params.ApplicationOfferAdminDetailsV4
```

Comment:
We know the size of the slice, we can allocate and assign elements without append()